xbmc/interfaces/builtins/GUIBuiltins.cpp

```diff
@@ -375,6 +375,16 @@ static int ToggleDirty(const std::vector<std::string>&)
   return 0;
 }
 
+/*! \brief Toggle display HDR On/Off (equivalent to Windows HDR switch in display settings).
+ *  \param params (ignored)
+ */
+static int ToggleDisplayHDR(const std::vector<std::string>&)
+{
+  CServiceBroker::GetWinSystem()->ToggleWindowsHDR();
```

Comment:
In another comment, i made mention of renaming this to use the base winsystem method setHDR(). This would be more inline with a generic platform agnostic method name in non platform code area

Reply:
Done